src/utils/utils_common.h

```diff
@@ -36,45 +35,36 @@ extern "C" {
 
 #define __TLS __declspec(thread)
 
-static inline char *util_getenv(const char *name) {
-    char *buffer;
-    size_t numberOfElements;
-    errno_t err = _dupenv_s(&buffer, &numberOfElements, name);
-    if (err) {
-        return NULL;
-    }
-
-    return buffer;
-}
-
-static inline void util_free_getenv(char *val) { free(val); }
-
 // TODO: implement util_get_page_size() for Windows
 static inline size_t util_get_page_size(void) { return 4096; }
 
 #else /* Linux */
 
 #define __TLS __thread
 
-static inline char *util_getenv(const char *name) { return getenv(name); }
-static inline void util_free_getenv(const char *val) {
-    (void)val; // unused
-}
-
 static inline size_t util_get_page_size(void) { return sysconf(_SC_PAGE_SIZE); }
 
 #endif /* _WIN32 */
 
+// Return value
+// If the function succeeds, the return value is the number of characters
+// stored in the buffer pointed to by buffer, not including
+// the terminating null character.
+//
+// If buffer is not large enough to hold the data, the return value
+// is (-1) * the buffer size (with the minus sign), in characters,
```

Comment:
I'm not sure if the part `(with the minus sign)` is required...?

Reply:
I'm not sure either. It is an explanation to be sure it is well described.

Reply:
Removed